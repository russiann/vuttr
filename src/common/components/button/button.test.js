import React from 'react';
import {render, cleanup} from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import Button from './button.component';

afterEach(cleanup);

it('renders', () => {
  const {asFragment} = render(<Button>Title</Button>);
  expect(asFragment()).toMatchSnapshot();
});

it('render text', () => {
  const text = 'Hello World!';
  const {getByTestId} = render(<Button>{text}</Button>);
  expect(getByTestId('button')).toHaveTextContent(text);
});
