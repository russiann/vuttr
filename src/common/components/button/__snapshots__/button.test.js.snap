// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <button
    class="css-rxzkzy"
    data-testid="button"
  >
    Title
  </button>
</DocumentFragment>
`;
